appium-dotnet-driver/Appium/AppiumDriver.cs

```diff
@@ -898,6 +901,19 @@ public void Zoom(IWebElement el)
 
         #endregion
 
+        #region Device Time
+

```

Comment:
add doc tags here.. would be good for people using this class to know what this property is used for.